Analysis/Tasks/PWGLF/spectraTPC_split.cxx

```diff
@@ -21,11 +20,20 @@ using namespace o2;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
 
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  std::vector<ConfigParamSpec> options{{"pikapr", VariantType::Int, 0, {"PiKaPr only"}}};
```

Comment:
We cannot change this on hyperloop. So it will not help us to assess the memory needs. I fear you need to make two executables...

Reply:
You are right, but that's ok, I will duplicate the code